src/object_cmd.h

```diff
@@ -14,7 +14,9 @@
 #include "object_type.h"
 
 CommandCost CmdBuildObject(DoCommandFlag flags, TileIndex tile, ObjectType type, uint8 view);
-
 DEF_CMD_TRAIT(CMD_BUILD_OBJECT, CmdBuildObject, CMD_DEITY | CMD_NO_WATER | CMD_AUTO, CMDT_LANDSCAPE_CONSTRUCTION)
 
+CommandCost CmdBuildObjectArea(DoCommandFlag flags, TileIndex tile, uint32 start_tile, ObjectType type, uint8 view, bool diagonal);
```

Comment:
Use `TileIndex` for `start_tile`.

Also, current _cmd.h files have all functions prototypes first and then all DEF_CMD_TRAIT.